script/release/notes/notes.js

```diff
@@ -6,10 +6,19 @@ const fs = require('fs');
 const path = require('path');
 
 const { GitProcess } = require('dugite');
-const { Octokit } = require('@octokit/rest');
-const octokit = new Octokit({
-  auth: process.env.ELECTRON_GITHUB_TOKEN
-});
+
+let octokit;
+if (!octokit) {
+  const octokitParams = { auth: process.env.ELECTRON_GITHUB_TOKEN };
+  try {
```

Comment:
😕  Why do we need this? This code will only run on the current branch, never across-branches

Reply:
Agreed ☝🏻  let's remove this, it should be whatever's necessary on that branch

Reply:
868f90e86bd2f79cdb7be0199d48246776cee38f